app/src/main/java/com/arygm/quickfix/ui/authentication/PasswordScreen.kt

```diff
@@ -188,3 +233,29 @@ fun PasswordScreen(navigationActions: NavigationActions) {
         })
   }
 }
+
+fun stringToTimestamp(birthDate: String): com.google.firebase.Timestamp? {
+  val dateParts = birthDate.split("/")
+  return if (dateParts.size == 3) {
+    val day = dateParts[0].toIntOrNull()
+    val month = dateParts[1].toIntOrNull()
+    val year = dateParts[2].toIntOrNull()
+    if (day != null && month != null && year != null) {
+      val calendar =
+          GregorianCalendar(year, month - 1, day).apply {
+            set(Calendar.HOUR_OF_DAY, 0)
+            set(Calendar.MINUTE, 0)
+            set(Calendar.SECOND, 0)
+            set(Calendar.MILLISECOND, 0)
+          }
+      val date = calendar.time
+      com.google.firebase.Timestamp(date)
+    } else {
+      Log.e("DateConversion", "Invalid date format: $birthDate")
+      null
+    }
+  } else {
+    Log.e("DateConversion", "Date string is not in the correct format: $birthDate")
+    null
+  }
+}
```

Comment:
This will not be used in Next PRs as we will combine `InfoScreen` & `PasswordScreen` in one screen (`RegisterScreen`)